src/networking.c

```diff
@@ -513,10 +513,14 @@ void afterErrorReply(client *c, const char *s, size_t len, int flags) {
     if (!(flags & ERR_REPLY_FLAG_NO_STATS_UPDATE)) {
         /* Increment the global error counter */
         server.stat_total_error_replies++;
-        /* Increment the error stats
-         * If the string already starts with "-..." then the error prefix
-         * is provided by the caller ( we limit the search to 32 chars). Otherwise we use "-ERR". */
-        if (s[0] != '-') {
+        /* Increment the error stats */
+        /* After the errors RAX reaches this limit, instead of tracking
+         * custom LUA errors, we track the error under `error_LUA` */
+        if (flags & ERR_REPLY_FLAG_LUA && raxSize(server.errors) >= ERROR_STATS_LUA_LIMIT) {
+            incrementErrorCount(LUA_ERRORSTATS_OVERFLOW_ERR, strlen(LUA_ERRORSTATS_OVERFLOW_ERR));
+        /* If the string already starts with "-..." then the error prefix
+        * is provided by the caller ( we limit the search to 32 chars). Otherwise we use "-ERR". */ 
```

Comment:
nit: this is an odd placement of comment. Should we move it inside the else if block?

Reply:
This comment was originally added above the `if` block.  With this new logic, it became an `else if` block.

I looked at examples of how we add comments for `else if` blocks and saw this styling was used: https://github.com/valkey-io/valkey/blob/d52c8f30e0b4730115ed1cd0cdf36637aca7fef7/src/aof.c#L1081-L1084

I could move this comment inside the block too - I am fine with either way

Reply:
Ok, if this style is used in other files, we can stick to it.